tutorials/jitsi-debian/index.mdx

```diff
@@ -57,9 +57,10 @@ The tool provides features like:
     systemctl enable nginx.service
     ```
 
-  <Message type="important">
+    <Message type="important">
     If Nginx or Apache are not installed, Jitsi Meet will automatically install [Jetty](https://www.eclipse.org/jetty/) during the installation.
```

Comment:
```suggestion
    If you have not installed Nginx or Apache, Jitsi Meet will automatically install [Jetty](https://www.eclipse.org/jetty/) during the installation.
```

Reply:
I'm not sure the indentation of the text inside the message box is correct, dit it display correctly when you did the build?

Reply:
Yes perfect!